HTML/mockup-mogo/sass/components/_offer-item.scss

```diff
@@ -0,0 +1,36 @@
+.offer-item {
+    @include flex($align: flex-start);
+    padding: 10% 5%;
+
+    &:nth-child(-n + 3) {
```

Comment:
just in case there is a lot cases when better to use classes instead, maybe it is good case for nth-child